src/alchemtest/amber/access.py

```diff
@@ -96,12 +94,57 @@ def load_invalidfiles():
         - 'data' : the example of invalid data files
         - 'DESCR': the full description of the dataset
 
+
+    .. deprecated:: 0.7
+       use :func:`load_testfiles` instead
+
     """
 
-    module_path = dirname(__file__)
-    data = [glob(join(module_path, 'invalidfiles/*.out.bz2'))]
+    import warnings  # NOTE: this import is here and not where it should be just because we'll remove this function in next version
+    warnings.warn(
+        "load_invalidfiles() was deprecated in 0.7.0 and will be removed in the following release."
+        " Use load_testfiles() instead",
+        DeprecationWarning)
+    module_path = Path(__file__).parent
+    data = [[
+        module_path / 'testfiles' / 'no_useful_data.out.tar.bz2',
+        module_path / 'testfiles' / 'no_control_data.out.tar.bz2',
+        module_path / 'testfiles' / 'no_temp0_setted.out.tar.bz2',
+        module_path / 'testfiles' / 'no_free_energy_info.out.tar.bz2',
+        module_path / 'testfiles' / 'no_atomic_section.out.tar.bz2',
+        module_path / 'testfiles' / 'no_results_section.out.tar.bz2']]
+
+    with open(module_path / 'testfiles' / 'descr_invalid.rst') as rst_file:
+        fdescr = rst_file.read()
+
+    return Bunch(data=data,
+                 DESCR=fdescr)
+
+
+def load_testfiles():
+    """Load incomplete or wrongly formatted files to be used to test the AMBER parsers.
+
+    Returns
+    -------
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+
+        - 'data' : the data files
+        - 'DESCR': the full description of all the files
+
+    """
+
+    testfiles_path = Path(__file__).parent / 'testfiles'
+
+    data = {}
+    for f in testfiles_path.iterdir():
+        f_path = str(f)
+        if f.suffix==".bz2":
+            while f.suffix in ('.tar', '.bz2', '.out'):
+                f = f.with_suffix('')
+            data[f.name] = [f_path]
```

Comment:
So what I was wondering is that 
if f has suffix of `'.tar'` or `'.out'`, they won't pass the `if f.suffix==".bz2":` in the first place?